docs/content/b2.md

```diff
@@ -172,11 +172,6 @@ the file instead of hiding it.
 Old versions of files, where available, are visible using the 
 `--b2-versions` flag.
 
-**NB** Note that `--b2-versions` does not work with crypt at the
```

Comment:
This feature should probably get a mention in the docs but not here!

Reply:
I'm not quite sure what you mean. Since this is merely a bugfix for B2 users, the removal of this notice seems appropriate? Where else might I change the docs?